package.json

```diff
@@ -27,34 +27,33 @@
     "all": "npm run lint && npm run format && npm run check && npm run build && npm t"
   },
   "devDependencies": {
-    "@playwright/test": "^1.27.1",
-    "@size-limit/file": "^8.1.0",
+    "@playwright/test": "^1.30.0",
+    "@size-limit/file": "^8.2.4",
     "@sveltejs/adapter-netlify": "next",
-    "@sveltejs/kit": "next",
-    "@types/html-to-text": "^8.1.1",
+    "@sveltejs/kit": "^1.7.1",
+    "@types/html-to-text": "^9.0.0",
     "@types/qrcode": "^1.5.0",
-    "@types/rss": "^0.0.29",
-    "@typescript-eslint/eslint-plugin": "^5.42.0",
-    "@typescript-eslint/parser": "^5.42.0",
-    "@vitest/coverage-c8": "^0.24.4",
-    "devalue": "^4.2.0",
-    "discord.js": "^13.6.0",
+    "@types/rss": "^0.0.30",
+    "@typescript-eslint/eslint-plugin": "^5.52.0",
+    "@typescript-eslint/parser": "^5.52.0",
+    "@vitest/coverage-c8": "^0.28.5",
+    "devalue": "^4.3.0",
+    "discord.js": "^14.7.1",
```

Comment:
For legacy purposes, the Discord.js dependency should remain at `v^13.6.0`.

Reply:
Attempted to resolve via `npm i -D discord.js@^13.6.0`.
 <https://github.com/EthanThatOneKid/acmcsuf.com/blob/b5e94546b96632b2955ac52838a323ca09b5a47d/package.json#L41>

Reply:
Closing #582 proved that `scripts/close-issue-channel.js` is still working. 

<img width="119" alt="image" src="https://user-images.githubusercontent.com/31261035/219825146-b2b5bfb3-338b-4ac4-aed5-d673abf46bd6.png">

To clean up our legacy open issue channels, we plan to close and reopen all of the issues such that any existing conversations get archived as intended via `scripts/close-issue-channel.js` and each thread gets reopened as new issue threads. To prepare for this, I created <https://github.com/EthanThatOneKid/gitcord/pull/23>.